src/libs/actions/App.ts

```diff
@@ -461,7 +461,7 @@ function openProfile(personalDetails: OnyxTypes.PersonalDetails) {
                 onyxMethod: Onyx.METHOD.MERGE,
                 key: ONYXKEYS.PERSONAL_DETAILS_LIST,
                 value: {
-                    [currentUserAccountID]: {
+                    [currentUserAccountID ?? '']: {
```

Comment:
I don't think it makes much sense to use an empty string key here's. I don't think it's possible to call openProfile without a currentUserAccountID being set. Can we assert that it will be a number?

Or if that doesn't work for some reason let's omit these updates entirely if it's null.

Reply:
@neil-marcellini Lint gave errors while using the assertion, so I added a more concrete check
```
 if (typeof currentUserAccountID === 'number') {
...
```